instrumentation/internal/internal-application-logger/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/logging/ApplicationLoggerBridge.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap.logging;
+
+import io.opentelemetry.javaagent.bootstrap.InternalLogger;
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class ApplicationLoggerBridge {
+
+  private static final AtomicReference<ApplicationLoggerBridge> applicationLoggerBridge =
+      new AtomicReference<>();
+
+  public static void set(ApplicationLoggerBridge bridge) {
+    applicationLoggerBridge.compareAndSet(null, bridge);
```

Comment:
can we throw exception if already set, or is that valid in some cases?

Reply:
I don't think it's valid; there is supposed to be only one bridge installed.
WDYT about printing out a WARN/ERROR instead?

Reply:
No, this should never happen in a properly constructed javaagent.